<<<<<<< HEAD
1. The system I used is the OU supercomputer, and additional software like R-studio and git hub. Our account on OSCER system was created by Dr. Toomy, and all work done was performed by activating the environment created by Dr. Toomey with following activation code:

```bach 

mamba activate /home/mbtoomey/.conda/envs/BIOL7263_Genomics
```

2. For the project, my hypothesis was whether long reads generated by oxford nanopore technology alone will be as qualitative or not as produced by illumina sequencing short reads if high base calling data from oxford nanopre used . Though the reads generated by nanopore sequencing have long reads but has higher error and low base sequence quality than comparison to short reads illumina sequencing. The hypothesis is that the use of high base calling data from oxford nanopore will generate qualitative data avoiding the need of illumina sequencing.  

For the project, I choose Campylobacter jejuni starin S2-20. The illumina data for the genome was obtained from European nucleotide archive with SRA accession numbers (SRX22060912) using link 

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_1.fastq.gz


wget  ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_2.fastq.gz
```



3. Oxford nanopore long reads were downloaded from NCBI with accession number (SRX22060913) using following link:

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/090/SRR26353490/SRR26353490_1.fastq.gz
```



4. Analysis: Quality of short reads from illumina was done using Fastqc tool with following fastqc .sbatch and .sh command:

-sbatch file:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --mem 8G
#SBATCH --output=camphylobacter_fastqc_%J_stdout.txt
#SBATCH --error=camphylobacter_fastqc_%J_stderr.txt
#SBATCH --job-name=camphylobacter_fastqc
# 

bash /home/biol726313/BIOL7263_Genomics/scripts/camphylobacter/camphylobacter_fastqc.sh
```


-sh file:

```bash

fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_1.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/
fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_2.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/
```

5. Adaptor was trimmed by using trim galore with following sbathch command:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=4
#SBATCH --mem 100G
#SBATCH --output=short_reads_trim_%J_stdout.txt
#SBATCH --error=short_reads_trim_%J_stderr.txt
#SBATCH --job-name=short_reads_trim
# 

bash trim_galore.sh

```

-and sh: 
```bach 
trim_galore --paired --fastqc --gzip --cores 4 --length 100 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_1.fastq.gz /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_2.fastq.gz --basename trimmed_reads -o /scratch/biol726313/BIOL7263_Genomics/camphylobacter/
```

6. Quality check of oxford nanopore reads was done by using tool nanoplot. Below are the .sbatch and .sh file. For using this tool a separate environment was created and run on separate environment.

```bach
#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=8
#SBATCH --mem 80G
#SBATCH --output=nanoplot_%J_stdout.txt
#SBATCH --error=nanoplot__%J_stderr.txt
#SBATCH --job-name=nanoplot
# 

bash nanoplot.sh

```

-and .sh:
```bach

NanoPlot --fastq /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/SRR26353490_1.fastq.gz --outdir /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/nanoplot_output --title "SRR26353490_1 Quality"
```
7. In the next step sequence obtained from nanopore were trimmed to remove adapter sequences using tool porechop. The sbatch and .sh script to run the tool are below:
```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=8
#SBATCH --mem 120G
#SBATCH --output=porechop_%J_stdout.txt
#SBATCH --error=porechop_%J_stderr.txt
#SBATCH --job-name=porechop
# 

bash porechop.sh
```

```bach
porechop -i /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/SRR26353490_1.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/SRR26353490_1.trim.fastq.gz
```
8. For the illumina trimmed sequence, spades was run separately for de novo assembly. The script for .sbatch and .sh are given below:

```bach
#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=24
#SBATCH --mem 100G
#SBATCH --output=full_spades_%J_stdout.txt
#SBATCH --error=full_spades_%J_stderr.txt
#SBATCH --job-name=full_sapdes
# 

bash short_reads_assembly.sh
```
```bach
spades.py --phred-offset 33 --threads 24 --careful -o /scratch/biol726313/BIOL7263_Genomics/camphylobacter/assembly \
-1 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/trimmed_reads_val_1.fq.gz \
-2 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/trimmed_reads_val_2.fq.gz
```
9. QC assembly is done for illumina sequence using quast with .sbatch and .sh batch script below:

```bach
#!/bin/bash
#
#SBATCH --partition=debug
#SBATCH --ntasks=1
#SBATCH --mem 20G
#SBATCH --output=quast_%J_stdout.txt
#SBATCH --error=quast_%J_stderr.txt
#SBATCH --job-name=quast
# 

bash quast.sh

```
```bach
quast.py --output-dir /scratch/biol726313/BIOL7263_Genomics/camphylobacter/assembly/quast /scratch/biol726313/BIOL7263_Genomics/camphylobacter/assembly/contigs.fasta
```
10. Trim sequenced from illumina and nanopore is de novo assembled by using spades hybrid. Below is the script for sbatch and .sh files.

```bach
#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=24
#SBATCH --mem 100G
#SBATCH --output=spades_hybrid_%J_stdout.txt
#SBATCH --error=spades_hybrid_%J_stderr.txt
#SBATCH --job-name=spades_hybrid
# 

bash spades_hybrid_assembly.sh
```
```bach
spades.py --phred-offset 33 --threads 24 --careful -1 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/trimmed_reads_val_1.fq.gz -2 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/trimmed_reads_val_2.fq.gz --nanopore /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/SRR26353490_1.trim.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/spades_hybrid_assembly/spades_hybrid_output

```
11. The hybrid assembly quality was checked by hybrid quast with following .sbatch and .sh script

```bach
#!/bin/bash
#
#SBATCH --partition=debug
#SBATCH --ntasks=1
#SBATCH --mem 20G
#SBATCH --output=quast_hybrid_%J_stdout.txt
#SBATCH --error=quast_hybrid_%J_stderr.txt
#SBATCH --job-name=quast_hybrid
# 

bash quast_hybrid.sh

```

```bach
quast.py --output-dir /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/spades_hybrid_assembly/quast /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/spades_hybrid_assembly/spades_hybrid_output/contigs.fasta

```

12. RESULT:
Per base sequence quality from illumina was 30, which was much higher than from the oxford nanopore (per base sequence quality was 14.2).However, there were much larger contigs from oxford nanopore, if combine hybrid analysis was done, the result obtained was great. The result obtained from hybrid spades was much better (10X) in terms of N50 than the sequence from ilumina used alone. So, combined sequence analysis from illumina and oxford nanopore is much better and no nucleotide mismatches per 100 Kbp.

# Fast QC report from illumina sequencing
![FastQC report from illumina](https://github.com/biol726313/mku8778/blob/main/SRR26353491_1.FASTQ.GZ.jpeg)

![FastQC report from illumina read2](https://github.com/biol726313/mku8778/blob/main/SRR26353491_2.fastq.gz.jpeg)

Below is the final report after quast analysis from illumina and hybrid.
# [pdf from illumina](https://github.com/biol726313/mku8778/blob/main/report.pdf)
# [pdf from hybrid](https://github.com/biol726313/mku8778/blob/main/report%20hybrid.pdf)

My next step would be to analyze the high accuracy base calling sequence data from oxford nanopore using Dorado and compare the quality of assembled sequence with illumina alone and hybrid from oxford nanopore sequence. If the quality of sequence data obtained from oxford nanopore would be identical, then process of sequencing genome using nanopore would be less time consuming and economical.

13. The experience i got in this seminar was great. I appreciate Dr. Toomey for his hard work, pushing me all the time when i need help. This seminar will be a key part in my PhD journey. 



