<<<<<<< HEAD
1. The system I used is the OU supercomputer, and additional software like R-studio and git hub. Our account on OSCER system was created by Dr. Toomy, and all work done was performed by activating the environment created by Dr. Toomey with following activation code:

```bach 

mamba activate /home/mbtoomey/.conda/envs/BIOL7263_Genomics
```

2. For the project, my interest was whether long reads generated by oxford nanopore technology alone will be as qualitative or not as produced by illumina sequencing short reads is high base calling data alone is used. Though the reads generated by nanopore sequencing have long reads but has higher error and low base sequence quality than comparison to short reads illumina sequencing. The hypothesis is that the use of high base calling data from oxford nanopore will generate qualitative data avoiding the need of illumina sequencing.  

For the project, I choose Campylobacter jejuni starin S2-20. The illumina data for the genome was obtained from European nucleotide archive with SRA accession numbers (SRX22060912) using link 

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_1.fastq.gz


wget  ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_2.fastq.gz
```



3. Oxford nanopore long reads were downloaded from NCBI using following link:

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/090/SRR26353490/SRR26353490_1.fastq.gz
```



4. Analysis: Quality of short reads from illumina was done using Fastqc tool with following fastqc .sbatch and .sh command:

-sbatch file:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --mem 8G
#SBATCH --output=camphylobacter_fastqc_%J_stdout.txt
#SBATCH --error=camphylobacter_fastqc_%J_stderr.txt
#SBATCH --job-name=camphylobacter_fastqc
# 

bash /home/biol726313/BIOL7263_Genomics/scripts/camphylobacter/camphylobacter_fastqc.sh
```


-sh file:

```bash

fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_1.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/
fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_2.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/
```

5. Adaptor was trimmed by using trim galore with following sbathch command:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=4
#SBATCH --mem 100G
#SBATCH --output=short_reads_trim_%J_stdout.txt
#SBATCH --error=short_reads_trim_%J_stderr.txt
#SBATCH --job-name=short_reads_trim
# 

bash trim_galore.sh

```

-and sh: 
```bach 
trim_galore --paired --fastqc --gzip --cores 4 --length 100 /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_1.fastq.gz /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_2.fastq.gz --basename trimmed_reads -o /scratch/biol726313/BIOL7263_Genomics/camphylobacter/
```

6. Quality of oxford nanopore reads was done by using tool nanoplot. Below are the .sbatch and .sh batch file. For using this tool a seperate environment was created and run on seperate environment.

```bach
#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=8
#SBATCH --mem 80G
#SBATCH --output=nanoplot_%J_stdout.txt
#SBATCH --error=nanoplot__%J_stderr.txt
#SBATCH --job-name=nanoplot
# 

bash nanoplot.sh

```

-and .sh:
```bach

NanoPlot --fastq /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/SRR26353490_1.fastq.gz --outdir /scratch/biol726313/BIOL7263_Genomics/ONT_longreads/nanoplot_output --title "SRR26353490_1 Quality"
```

