<<<<<<< HEAD
1. The system I used is the OU supercomputer, and additional software like R-studio and git hub. Our account on OSCER system was created by Dr. Toomy, and all work done was performed by activating the environment created by Dr. Toomey with following activation code:

```bach 

mamba activate /home/mbtoomey/.conda/envs/BIOL7263_Genomics
```

2. For the project, my interest was whether long reads generated by oxford nanopore technology alone will be as qualitative or not as produced by illumina sequencing short reads. Because the reads generated by nanopore sequencing have the higher error than comparison to short reads illumina sequencing. The hypothesis is that the use of high base calling data from oxford nanopore will generate qualitative data avoiding the need of illumina sequencing.  

For the project, I choose Campylobacter jejuni starin S2-20. The illumina data for the genome was obtained from European nucleotide archive with SRA accession numbers (SRX22060912) using link 

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_1.fastq.gz


wget  ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_2.fastq.gz


3. Oxford nanopore long reads were downloaded from NCBI using following link:

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/090/SRR26353490/SRR26353490_1.fastq.gz


4. Analysis: Fast Qc of the short raeds from illumina was done with following fastqc command:

-sbatch file:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=1
#SBATCH --mem 8G
#SBATCH --output=camphylobacter_fastqc_%J_stdout.txt
#SBATCH --error=camphylobacter_fastqc_%J_stderr.txt
#SBATCH --job-name=camphylobacter_fastqc
# 

bash /home/biol726313/BIOL7263_Genomics/scripts/camphylobacter/camphylobacter_fastqc.sh


-sh file:

```bash

fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_1.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/
fastqc /scratch/biol726313/BIOL7263_Genomics/camphylobacter/SRR26353491_2.fastq.gz -o /scratch/biol726313/BIOL7263_Genomics/camphylobacterfastqc_output/


5. To trim adaptor sequence, trim galore was used with following sbathch command:

```bach

#!/bin/bash
#
#SBATCH --partition=normal
#SBATCH --ntasks=4
#SBATCH --mem 100G
#SBATCH --output=short_reads_trim_%J_stdout.txt
#SBATCH --error=short_reads_trim_%J_stderr.txt
#SBATCH --job-name=short_reads_trim
# 

bash trim_galore.sh


-and sh 
=======
1. The system I used is the OU supercomputer, and additional software like R-studio and git hub. Our account on OSCER system was created by Dr. Toomy, and all work done was performed by activating the environment created by Dr. Toomey with following activation code:

```bach 

mamba activate /home/mbtoomey/.conda/envs/BIOL7263_Genomics
```

2. For the project, my interest was whether long reads generated by oxford nanopore technology alone will be as qualitative or not as produced by illumina sequencing short reads. Because the reads generated by nanopore sequencing have the higher error than comparison to short reads illumina sequencing. The hypothesis is that the use of high base calling data from oxford nanopore will generate qualitative data avoiding the need of illumina sequencing.  

For the project, I choose Campylobacter jejuni starin S2-20. The illumina data for the genome was obtained from European nucleotide archive with SRA accession numbers (SRX22060912) using link 

```bach

wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_1.fastq.gz

and

```bach

wget  ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR263/091/SRR26353491/SRR26353491_2.fastq.gz
>>>>>>> c5a852eeb8787caeba53cb42991205c746b09702
