# My BIOL726313 portfolio

This is the home for my class project!

Here are my notes from:
[Day 1](https://github.com/biol726313/mku8778/edit/main/day1.md)
[Day 2](https://github.com/biol726313/mku8778/edit/main/day2.md)

Class project: 
[Research proposal](https://github.com/biol726313/mku8778/blob/main/class%20project.pdf)

## fastQc files:

[ecoli_fastqc.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_fastqc.sh)

[ecoli_fastqc.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_fastqc.sbatch)

## Running trim glore:

[ecoli_trim.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_trim.sh)

[ecoli_trim.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_fastqc.sbatch)

## Generating index:

[ecoli_index.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_index.sh)

[ecoli_index.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_index.sbatch)


## Maping
[ecoli_bwa_mem.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_bwa_mem.sh)

[ecoli_bwa_mem.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_bwa_mem.sbatch)

## Converting SAM to BAM

[ecoli_samtools_view.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_samtools_view.sh)

[ecoli_samtools_view.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_samtools_view.sbatch)

## Sorting a BAM File
[ecoli_map_sort.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_map_sort.sh)

[ecoli_map_sort.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_map_sort.sbatch)

## Removing suspected duplicates

[ecoli_markdup.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_markdup.sh)

[ecoli_markdup.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_markdup.sbatch)

## Automated Analyses

[ecoli_vcf.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_vcf.sh)

[ecoli_vcf.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_vcf.sbatch)


[ecoli_call.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_call.sh)

[ecoli_call.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_call.sbatch)

[ecoli_filt.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_filt.sh)

[ecoli_filt.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_filt.sbatch)

## Locating Genes that are Missing Compared to the Reference

[ecoli_cover.sh](https://github.com/biol726313/mku8778/blob/main/ecoli_cover.sh)

[ecoli_cover.sbatch](https://github.com/biol726313/mku8778/blob/main/ecoli_cover.sbatch)

## Assembly of Unmapped Reads

[bam_to_fasta.sh](https://github.com/biol726313/mku8778/blob/main/bam_to_fasta.sh)

[bam_to_fasta.sbatch](https://github.com/biol726313/mku8778/blob/main/bam_to_fasta.sbatch)

## De-novo Assembly

[unmapped_spades.sh](https://github.com/biol726313/mku8778/blob/main/unmapped_spades.sh)

[unmapped_spades.sbatch](https://github.com/biol726313/mku8778/blob/main/unmapped_spades.sbatch)

## Assesment of Assemblies

[unmapped_quast.sh](https://github.com/biol726313/mku8778/blob/main/unmapped_quast.sh)

[unmapped_quast.sbatch](https://github.com/biol726313/mku8778/blob/main/unmapped_quast.sbatch)

## Obtain Open Reading Frames

[unmapped_orf.sh](https://github.com/biol726313/mku8778/blob/main/unmapped_orf.sh)

[unmapped_orf.sbatch](https://github.com/biol726313/mku8778/blob/main/unmapped_orf.sbatch)

## Additional Checks

[unmapped_blastn.sh](https://github.com/biol726313/mku8778/blob/main/unmapped_blastn.sh)

[unmapped_blastn.sbatch](https://github.com/biol726313/mku8778/blob/main/unmapped_blastn.sbatch)

## Run Open Reading Frames Through pfam_scan

[unmapped_pfam.sh](https://github.com/biol726313/mku8778/blob/main/unmapped_pfam.sh)

[unmapped_pfam.sbatch](https://github.com/biol726313/mku8778/blob/main/unmapped_pfam.sbatch)

## De novo Assembly Using Short Reads

[full_spades.sh](https://github.com/biol726313/mku8778/blob/main/full_spades.sh)

[full_spades.sbatch](https://github.com/biol726313/mku8778/blob/main/full_spades.bash)

## checking assembly

[quast.sh](https://github.com/biol726313/mku8778/blob/main/quast.sh)

[quast.sbatch](https://github.com/biol726313/mku8778/blob/main/quast.sbatch)

## Index the contigs

[reads_index.sh](https://github.com/biol726313/mku8778/blob/main/reads_index.sh)

[reads_index.sbatch](https://github.com/biol726313/mku8778/blob/main/reads_index.sbatch)

## align QC reads to contigs and output SAM file

[mapping.sh](https://github.com/biol726313/mku8778/blob/main/mapping.sh)

[mapping.sbatch](https://github.com/biol726313/mku8778/blob/main/mapping.sbatch)

## convert the SAM file to a BAM file

[sam_tobam.sh](https://github.com/biol726313/mku8778/blob/main/sam_tobam.sh)

[sam_tobam.sbatch](https://github.com/biol726313/mku8778/blob/main/sam_tobam.sbatch)

## sort the BAM file

[bam_sort.sh](https://github.com/biol726313/mku8778/blob/main/bam_sort.sh)

[bam_sort.sbatch](https://github.com/biol726313/mku8778/blob/main/bam_sort.sbatch)

## Index BAM

[index_bam.sh](https://github.com/biol726313/mku8778/blob/main/index_bam.sh)

[index_bam.sbatch](https://github.com/biol726313/mku8778/blob/main/index_bam.sbatch)

## Mapping statstistics

[qualimap.sh](https://github.com/biol726313/mku8778/blob/main/qualimap.sh)

[qualimap.sbatch](https://github.com/biol726313/mku8778/blob/main/qualimap.sbatch)

## BLAST search

[blast.sh](https://github.com/biol726313/mku8778/blob/main/blast.sh)

[blast.sbatch](https://github.com/biol726313/mku8778/blob/main/blast.sbatch)

## Annotation of de novo Assembled Contigs

[orfipy.sh](https://github.com/biol726313/mku8778/blob/main/orfipy.sh)

[orfipy.sbatch](https://github.com/biol726313/mku8778/blob/main/orfipy.sbatch)

[pfam.sh](https://github.com/biol726313/mku8778/blob/main/pfam.sh)

[pfam.sbatch](https://github.com/biol726313/mku8778/blob/main/pfam.sbatch)

[blast_p.sh](https://github.com/biol726313/mku8778/blob/main/blast_p.sh)

[blast_p.sbatch](https://github.com/biol726313/mku8778/blob/main/blast_p.sbatch)

## Hybrid de novo Assembly

## QC the data

[pseudo_fastqc.sh](https://github.com/biol726313/mku8778/blob/main/pseudo_fastqc.sh)

[pseudo_fastqc.sbatch](https://github.com/biol726313/mku8778/blob/main/pseudo_fastqc.sbatch)

## Trim the Reads

[pseudo_trim.sh](https://github.com/biol726313/mku8778/blob/main/pseudo_trim.sh)

[pseudo_trim.sbatch](https://github.com/biol726313/mku8778/blob/main/pseudo_trim.sbatch)

## Assembly

[pseudo_short_assembly.sh](https://github.com/biol726313/mku8778/blob/main/pseud_short_assembly.sh)

[pseudo_short_assembly.sbatch](https://github.com/biol726313/mku8778/blob/main/pseud_short_assembly.sbatch)

## Hybrid Assembly

[pseudo_long_assembly.sh](https://github.com/biol726313/mku8778/blob/main/pseudo_long_assembly.sh)

[pseudo_long_assembly.sbatch](https://github.com/biol726313/mku8778/blob/main/pseudo_long_assembly.sbatch)

## Align Reads Back to Reference

## Index the BAM file

[index_bam.sh](https://github.com/biol726313/mku8778/blob/main/index_bam.sh)

[index_bam.sbatch](https://github.com/biol726313/mku8778/blob/main/index_bam.sbatch)

## Mapping the Illumina reads

[illumina_map.sh](https://github.com/biol726313/mku8778/blob/main/illumina_map.sh)

[illumina_map.sbatch](https://github.com/biol726313/mku8778/blob/main/illumina_map.sbatch)

## Mapping the PacBio data to the assembly

[pacbio_map.sh](https://github.com/biol726313/mku8778/blob/main/pacbio_map.sh)

[pacbio_map.sbatch](https://github.com/biol726313/mku8778/blob/main/pacbio_map.sbatch)



